src/main/java/com/autotune/common/data/dataSourceQueries/DataSourceQueries.java

```diff
@@ -0,0 +1,24 @@
+package com.autotune.common.data.dataSourceQueries;
+
+/**
+ * This class contains PromQL queries as enum constants for various metrics related to Kubernetes clusters.
+ * TODO - Add a custom PromQL query to fetch Cluster info
+ *      - Refactor PromQL queries into a separate YAML file utilizing Custom Resource Definitions (CRD).
+ */
+public class DataSourceQueries {
+    public enum PromQLQuery {
+        CLUSTER_QUERY("query"),
```

Comment:
This is dummy right? Lets not add it if it is a dummy query

Reply:
yes, this is dummy placeholder for the query, will exclude it